source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -722,12 +722,26 @@ export class TextBlock extends CardElement {
                     break;
             }
 
-            // Looks like 1.33 is the magic number to compute line-height
-            // from font size.
-            this._computedLineHeight = fontSize * 1.33;
```

Comment:
No, we can't remove that. lineHeights in HostConfig isn't a documented feature, things need to continue working exactly as before in the absence of the lineHeights definition:
- lineHeights in HostConfig should be optional
- When specified, lineHeights must be fully defined, e.g. Small, Medium, etc. are all required
- When lineHeights isn't defined in HostConfig, line height should be computed exactly as before to maintain 100% backward compatibility

Reply:
have taken this aspects in latest commit